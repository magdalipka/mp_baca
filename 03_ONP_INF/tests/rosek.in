10
ONP: xabc**=
ONP: ab+a~a-+
INF: x=~~a+b*c
INF: t=~a<x<~b
INF: ( a,+ b)/..[c3
ONP: ( a,b,.).c;-,*
ONP: abc++def++g+++
INF: x=a=b=c^d^e
INF: (r+y)=a=(b+c)+d
INF: x=!(c>a & c<b)
