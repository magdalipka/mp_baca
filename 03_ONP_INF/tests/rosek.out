INF: x = a * ( b * c )
INF: a + b + ( ~ a - a )
ONP: x a ~ ~ b c * + =
ONP: t a ~ x < b ~ < =
ONP: a b + c /
INF: a * ( b - c )
INF: error
ONP: x a b c d e ^ ^ = = =
ONP: r y + a b c + d + = =
ONP: x c a > c b < & ! =
